develop-guide/writeblitz/writeblitz.rst

```diff
@@ -771,6 +771,40 @@ Below you can see an `example` of this.
         exclude:
             - 9999
 
+Verification of actions list outputs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is also possible to check and see if certain items exist within a output that is list. 
+
+.. code-block:: YAML
+
+  - api:
+      device: PE1
+      function: get_list_items
+      arguments:
+          name: [1,2,3,4,5,6,7]       # the output is [1,2,3,4,5,6,7]
+      include:
+          - 5                         # checks if 5 is in the list
+          - "6"                       # checks if 6 is in the list
```

Comment:
if "6"   as a string  no? compared to an int?

Reply:
it will cast the type to int, just wanted to show that